modules/cassandra/src/main/java/org/testcontainers/containers/delegate/CassandraDatabaseDelegate.java

```diff
@@ -0,0 +1,65 @@
+package org.testcontainers.containers.delegate;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.exceptions.DriverException;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.CassandraContainer;
+import org.testcontainers.delegate.AbstractDatabaseDelegate;
+import org.testcontainers.exception.ConnectionCreationException;
+import org.testcontainers.ext.ScriptUtils;
+
+import static org.testcontainers.containers.CassandraContainer.CQL_PORT;
+
+/**
+ * Cassandra database delegate
+ *
+ * @author Eugeny Karpov
+ */
+@Slf4j
+public class CassandraDatabaseDelegate extends AbstractDatabaseDelegate<Session> {
+
+    private CassandraContainer container;
+
+    public CassandraDatabaseDelegate(CassandraContainer container) {
+        this.container = container;
+    }
+
+    @Override
+    protected Session createNewConnection() {
+        try {
+            return Cluster.builder()
+                    .addContactPoint(container.getContainerIpAddress())
+                    .withPort(container.getMappedPort(CQL_PORT))
```

Comment:
WDYT about adding `cassandraContainer.getCluster()` method? So that consumers will use something like:
```java
return cassandraContainer.getCluster().newSession();
```

And if somebody extends the C* class and change the CQL port, it will still work

Reply:
That's actually a really good idea. I used to initialize Datastax datasource in my projects using system variables, but this allow to don't create them at all, if Cassandra is needed only in tests and not in the main configuration.
So it's a good alternative to system variables in tests. Done, thanks!